#
#

bool frontRearImpacted
bool sideImpacted
uint32 impactBitmap
